atividade03/src/main/java/br/edu/uepb/eda/atividade03/FilaComPilhas.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
+ */
+package br.edu.uepb.eda.atividade03;
+
+/**
+ *
+ * @author jeffgabriiel
+ */
+
+public class FilaComPilhas implements Fila_IF {
+
+    private PilhaComLista pilha_1;
+    private PilhaComLista pilha_2;
+
+    public FilaComPilhas() {
+        this.pilha_1 = new PilhaComLista();
+        this.pilha_2 = new PilhaComLista();
+    }
+
+   @Override
+	public void enqueue(Integer element) throws Exception {
+		if (isFull()) {
+			throw new Exception("Fila está cheia.");
+		} else {
+			while(!pilha_1.isEmpty()) {
+				pilha_2.push(pilha_1.pop());
+			}
+			pilha_2.push(element);
+			while(!pilha_2.isEmpty()) {
+				pilha_1.push(pilha_2.pop());
+			}
+		}
+	}
+
+    @Override
+    public Integer dequeue() throws Exception {
+        if (isEmpty()) {
+            throw new Exception("Não é possível remover (está vazia).");
+        }else {
+			int dequeued = pilha_1.pop();
```

Comment:
```suggestion
			return pilha_1.pop();
```

para simplificar poderia retornar diretamente o valor do retorno do `.pop()`